development-updates.md

```diff
@@ -30,6 +33,15 @@ Development updates from [`pipermerriam`](https://github.com/pipermerriam/)
 
 This is simply a placeholder for now.
 
+## Himanshu
+
+Development updates from [`himanshu`](https://github.com/himanshu3w/)
+
+### Himanshu: Update 1
+
+Researching : BeaconChain And L2 Scaling
```

Comment:
Maybe hold off on this until you have a bit more to say.  Simply listing topics is something you can do in your `./notes/` document.  Development updates should aim to be interesting to an external consumer.

Reply:
Working on same